roles/openshift_health_checker/openshift_checks/fluentd_config.py

```diff
@@ -0,0 +1,188 @@
+                      '"/etc/docker/daemon.json":\n\n'
+                      '{{ "log-driver": "{driver}" }}\n\n'
+                      'See https://docs.docker.com/engine/admin/logging/json-file '
+                      'for more information.').format(driver=recommended_logging_driver)
+
+        return error
+
+    def check_fluentd_env_var(self, task_vars):
+        """Read and return the value of the 'USE_JOURNAL' environment variable on a fluentd pod."""
+        running_pods = self.running_fluentd_pods(task_vars)
+        pod_containers = running_pods[0]["spec"]["containers"]
+
+        if not pod_containers:
+            msg = ('There are no running containers on selected Fluentd pod "{}".\n'
+                   'Unable to calculate expected logging driver.').format(running_pods[0]["metadata"]["name"])
+            raise OpenShiftCheckException(msg)
+
+        pod_env = pod_containers[0]["env"]
+        if not pod_env:
+            msg = ('There are no environment variables set on the Fluentd container "{}".\n'
+                   'Unable to calculate expected logging driver.').format(pod_containers[0]["name"])
+            raise OpenShiftCheckException(msg)
+
+        for env in pod_env:
+            if env["name"] == "USE_JOURNAL":
+                return env["value"] != "false"
+
+        return False
+
+    def running_fluentd_pods(self, task_vars):
+        """Return a list of running fluentd pods."""
+        fluentd_pods, error = self.get_pods_for_component(
+            self.execute_module,
+            self.logging_namespace,
+            "fluentd",
+            task_vars,
+        )
+        if error:
+            msg = 'Unable to retrieve any pods for the "fluentd" logging component: {}'.format(str(error))
+            raise OpenShiftCheckException(msg)
+
+        running_fluentd_pods = [pod for pod in fluentd_pods if pod['status']['phase'] == 'Running']
+        if not len(running_fluentd_pods):
+            msg = ('No Fluentd pods were found to be in the "Running" state. '
+                   'At least one Fluentd pod is required in order to perform this check.')
+
+            raise OpenShiftCheckException(msg)
+
+        return running_fluentd_pods
+
+    def get_pods_for_component(self, execute_module, namespace, logging_component, task_vars):
+        """Get all pods for a given component. Returns: list of pods for component, error string."""
+        try:
+            pod_output = self.exec_oc(
+                execute_module,
+                namespace,
+                "get pods -l component={} -o json".format(logging_component),
+                [],
+                task_vars
+            )
+        except OpenShiftCheckException as err:
+            if "false" not in str(err):
+                raise err
+
+        try:
+            pods = json.loads(pod_output)
+            if not pods or not pods.get('items'):
+                raise ValueError()
+        except ValueError:
```

Comment:
Suggestion:

```python
try:
    pods = json.loads(pod_output)
    assert pods and 'items' in pods
except (ValueError, TypeError, AssertionError):
    return ...
```

Though logically equivalent, I believe it is easier to reason about `A and B`, than `not (not A or not B)`?